<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Customers</class>
 <widget class="QWidget" name="Customers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="bill_master_resources.qrc">:/icons/icons/small_user_icon.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Customers</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>Search</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Search by customer</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="customerName"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="searchBtn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
            <property name="icon">
             <iconset resource="bill_master_resources.qrc">
              <normaloff>:/icons/icons/1434133405_search-icon-tm.png</normaloff>:/icons/icons/1434133405_search-icon-tm.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QToolButton" name="addCustomer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="bill_master_resources.qrc">
         <normaloff>:/icons/icons/1428360733_user_add.png</normaloff>:/icons/icons/1428360733_user_add.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QToolButton" name="editCustomer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset resource="bill_master_resources.qrc">
         <normaloff>:/icons/icons/1428360743_file_edit.png</normaloff>:/icons/icons/1428360743_file_edit.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QToolButton" name="deleteCustomer">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="bill_master_resources.qrc">
         <normaloff>:/icons/icons/1428360740_user_delete.png</normaloff>:/icons/icons/1428360740_user_delete.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="customersDetailsTable">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>Serial No.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Customer Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Email</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Address</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Phone</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Serial</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Comments</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="bill_master_resources.qrc"/>
 </resources>
 <connections/>
</ui>
